### PR TITLE
[2.3] [FrameworkBundle][Translation] allow multiple bundles in TranslationUpdateCommand

### PR DESCRIPTION
**Pull request by havvg**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | yes |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| License | MIT |
| Doc PR | to be done |
- add ability to update more than one bundle once
- add options to include app-wide data
- [ ] Update documentation on this command

### CODE REVIEW

File: src/Symfony/Bundle/FrameworkBundle/Command/TranslationUpdateCommand.php [hunk 1]
```diff
@@ -41,23 +45,47 @@ protected function configure()
             ->setName('translation:update')
             ->setDefinition(array(
                 new InputArgument('locale', InputArgument::REQUIRED, 'The locale'),
-                new InputArgument('bundle', InputArgument::REQUIRED, 'The bundle where to load the messages'),
+                new InputArgument(
+                    'bundle', InputArgument::OPTIONAL ^ InputArgument::IS_ARRAY,
```
Comment (OskarStark) [inline src/Symfony/Bundle/FrameworkBundle/Command/TranslationUpdateCommand.php]: wouldn't it be better to use 2 options ?

```bundle``` or ```bundles``` and handle them directly as array or not.
if both options are not provided **all** bundles were updated.

File: src/Symfony/Bundle/FrameworkBundle/Command/TranslationUpdateCommand.php [hunk 2]
```diff
@@ -41,23 +45,47 @@ protected function configure()
...
+                new InputOption(
+                    'output-dir', null, InputOption::VALUE_REQUIRED,
+                    'Directory to write the translation files into'
+                ),
+                new InputOption(
+                    'output-format', null, InputOption::VALUE_OPTIONAL,
+                    'Override the default output format', 'yml'
+                ),
+                new InputOption(
+                    'all-bundles', null, InputOption::VALUE_NONE,
```
Comment (OskarStark) [inline src/Symfony/Bundle/FrameworkBundle/Command/TranslationUpdateCommand.php]: this option is not necessary if you will have a look at my previous comment

**Comment by OskarStark** [comment]
i like this feature

@aitboudad can you please restart the travis job, i would say the error is not realted

**Comment by aitboudad** [comment]
It needs to rebase onto the 2.8 branch

**Comment by fabpot** [comment]
Closing as there is no activity. Anyone willing to take over this one would need to reopen a new PR.

**Pull request closed without merging**